src/components/RightSidebar/BreakoutRooms/BreakoutRoomItem.vue

```diff
@@ -187,10 +195,12 @@ export default {
 				})
 			} else {
 				try {
-					await this.$store.dispatch('switchToBreakoutRoomAction', {
-						token: this.$store.getters.parentRoomToken(this.roomToken),
-						target: this.roomToken,
-					})
+					if (this.mainConversation.breakoutRoomMode === CONVERSATION.BREAKOUT_ROOM_MODE.FREE) {
```

Comment:
```suggestion
					if (this.mainConversation.breakoutRoomMode !== CONVERSATION.BREAKOUT_ROOM_MODE.FREE) {
					return
			}
```

Just a visual preference to avoid block nesting